amftrack/util/image_analysis.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+from typing import Tuple, List
+
+
+def convert_to_micrometer(pixel_length, magnification=2):
+    """
+    Converts pixels into micrometers, based on the magnification of the microscope.
+    """
+    camera_res = 3.45
```

Comment:
shouldn't camera_res be an argument?

Reply:
Ouai, justement je me demandais où coder les variables de ce type.
Je voulais pas la mettre en argument pasque le but c'est d'avoir une fonction pratique avec peu d'arguments et utilisable facilement. Mais ce serait bien d'avoir un endroit où on met les paramètres généraux. Je veux dire les paramètres qui sont les même partout dans le repo, mais pour pas qu'ils soient hard codé si ils viennent à changer un jour.
Par exemple:
camera_res
dimension des images de prince (3600*4200 ou je sais plus)

Reply:
Oui, bonne idée. J'ai fait des trucs un peu dégueu avec le `pixel_size_factor` que j'ai parfois hardcodé dans des fonctions. Je crois que j'avais refactor à un moment et que je l'avais mis dans un util ou quelque chose de ce genre et que je l'importais.

Reply:
Oui ça me parait une bonne solutions pour les fonctions qu'on risque d'utiliser dans d'autres contextes.
Et pour les fonctions qu'on risque pas d'utiliser dans un autre contexte, on peut juste importer le paramètre et l'utiliser comme variable global dans la fonction sans l'ajouter au paramètres.